beacon-node-oapi.yaml

```diff
@@ -161,6 +161,8 @@ paths:
     $ref: "./apis/validator/sync_committee_contribution_and_proof.yaml"
   /eth/v1/validator/prepare_beacon_proposer:
     $ref: "./apis/validator/prepare_beacon_proposer.yaml"
+  /eth/v1/validator/register_validator:
```

Comment:
this may be splitting hairs but this *feels* like it belongs under the `builder` namespace

even if the beacon node is the one implementing it.

I don't feel too strongly about this but I think its worth considering

Reply:
I don't have a preference here. Will keep it as-is until we hear more feedback.

Reply:
I wouldn't change the namespace. These are for the validator client (the thing that has the keys) to call. 

A validator has many things it's trying to accomplish -- sign blocks, sign attestations, register duties, send signals to builders